design-system/src/theme-provider.tsx

```diff
@@ -67,6 +67,7 @@ type ThemeProviderProps = {
 };
 
 const ThemeProvider = (props: ThemeProviderProps) => {
+  const themeName = props.theme || 'default'; // Use 'default' theme if none is provided
```

Comment:
Not needed, as the value is defined in  React's `defaultProp` object.

Reply:
Updated in this commit: 3909ff9415b69ef3d077becfb386651d4400d852

Thanks!